### PR TITLE
fix version download queryset

### LINKED ISSUE #932: Download QGIS 3.2.0 as rst broken/raises 500 error
If you download rst for QGIS 3.2.0 you get a 500.

If I do the same in my dev env, I see:

```
MultipleObjectsReturned at /en/qgis/version/3.2.0/download/
get() returned more than one Version -- it returned 2!
```
And furter in the debug info:

selection_476

which to me looks like there is not enough info to select ONLY QGIS (as InaSAFE 3.2.0 is also coming though)?

Any pointers to fix this are appreciated

### PR DESCRIPTION
**Pull request by ann26**
fix #932 

This PR filters queryset using project and slug so it won't raise conflicts other projects.

### CODE REVIEW

File: django_project/changes/views/version.py [hunk 1]
```diff
@@ -740,6 +740,31 @@ def _prepare_zip_archive(self, document, version_obj):
 
         return temp_path
 
+    def get_queryset(self):
+        """Get the queryset for download.
+
```
Comment (ismailsunni) [inline django_project/changes/views/version.py]: __[Flake8]__

[W293] blank line contains whitespace

**Comment by timlinux** [comment]
Can you fix the failing test so we can merge this @ann26 ?

**Comment by ann26** [comment]
hi @timlinux I made the test in travis passed but I am still struggling to increase the coverage

**Comment by ann26** [comment]
hi @timlinux the tests are all passing! (finally!) 
could you take a final look at this please?

**Comment by timlinux** [comment]
LGTM thanks @ann26

**Pull request merged**